mlir/lib/Quantum/Utils/RemoveQuantum.cpp

```diff
@@ -59,6 +60,39 @@ void removeQuantumMeasurements(func::FuncOp &function, PatternRewriter &rewriter
     }
 }
 
+void replaceQuantumMeasurements(func::FuncOp &function, PatternRewriter &rewriter)
+{
+    function.walk([&](MeasurementProcess op) {
+        auto types = op->getResults().getTypes();
+        auto loc = op.getLoc();
+        std::vector<Value> results;
```

Comment:
Nice 👍 Just a small suggestion when working with LLVM/MLIR, use SmallVectors etc wherever possible: 
```suggestion
        SmallVector<Value> results;
```